rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.aggregation/230_composite.yml

```diff
@@ -490,6 +490,61 @@ setup:
   - match: { aggregations.test.buckets.0.doc_count: 1 }
 
 ---
+"Composite aggregation with invalid format":
+  - skip:
+      version: " - 7.99.99"
+      reason:  After key parse checking added in 7.14
```

Comment:
I often add words like "to be backported" so that its super obvious if someone finds this before the backport lands that it'll be changing and isn't just a mistake.